### ISSUE TITLE
[Feature] Run safari and chrome serial test cases in sequence one after the other.

### ISSUE THREAD

**Issue by InduKrish** [microsoft/playwright#24373]
### System info
- Playwright Version: 1.32.2
- Operating System: MAC
- Browser: [All, Chromium, Firefox, WebKit]
- Other info:

### Source code

Hi All, quick question, I have 10 tests that you want to run in chrome browser sequentially - i run them using serial mode. , so they run in sequence. now i want these 10 questions first to run in chrome, then the same test 10 runs in safari in sequence, but Safari execution should start only after all 10 tests are done executed in chrome…

if i have browser chrome, safari in playwright config file, will it run 10 tests in chrome and safari at the same time? can you please clarify how i can run them in sequence, safari tests should run only after chrome or vice versa

**Comment by pavelfeldman** [comment]
You could use project dependencies and make Safari project depend on Chrome, however it'll have a side effect of not running Safari if some of the Chrome tests fail.

**Comment by pavelfeldman** [comment]
Closing as per above, please feel free to open a new issue if this does not cover your use case.
